engine/requirements.txt

```diff
@@ -10,19 +10,19 @@ redis==3.4.1
 humanize==0.5.1
 uwsgi==2.0.21
 django-cors-headers==3.7.0
-django-debug-toolbar==3.2.1
+django-debug-toolbar==4.1
 django-sns-view==0.1.2
-python-telegram-bot==13.13
+python-telegram-bot==20.0a3
 django-silk==5.0.3
 django-redis-cache==3.0.0
 hiredis==1.0.0
 django-ratelimit==2.0.0
 django-filter==2.4.0
 icalendar==4.0.7
 recurring-ical-events==0.1.16b0
-slack-export-viewer==1.0.0
+slack-export-viewer==1.1.4
 beautifulsoup4==4.12.2
-social-auth-app-django==3.1.0
```

Comment:
updated this package for [this](https://github.com/python-social-auth/social-app-django/blob/8d0a2052d1b22a899454571c62237d23aa25af97/CHANGELOG.md?plain=1#L27)

```bash
apps/twilioapp/tests/test_phone_calls.py::test_forbidden_requests
  /usr/local/lib/python3.11/site-packages/social_django/urls.py:15: RemovedInDjango40Warning: django.conf.urls.url() is deprecated in favor of django.urls.re_path().
    url(r'^login/(?P<backend>[^/]+){0}$'.format(extra), views.auth,
```